Automattic/jetpack
# Extended Blocks: add Loom as oEmbed provider

This PR adds loom.com as a new oembed provider, allowing embed resources from this service.

#### Changes proposed in this Pull Request:

* Add loom.com as a new oembed provider

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Edit/Add a post
* Confirm you see the `Loom` variation among block options.

<img width="500" alt="Screen Shot 2020-09-10 at 9 28 43 PM" src="https://user-images.githubusercontent.com/77539/92831384-b849f300-f3ac-11ea-85ed-684cd52e8e9d.png">

* Add Loom embed variation to the editor. You can use this link https://www.loom.com/share/e3dcec661c37487b818b8e3b8225ec27

<img src="https://user-images.githubusercontent.com/77539/92831735-2098d480-f3ad-11ea-82a0-b8055a0e13c5.png" width="500px" />

* Finally, confirm the resource is properly rendered.

![image](https://user-images.githubusercontent.com/77539/92801481-0bf81480-f38c-11ea-83df-383ab03f4675.png)

* Confirm, you can paste the resource and it will be automatically transformed in a `core/embed` block, `Loom` variation.

Confirm it works with WordPress greater and lower than 5.5

<img width="400" src="https://user-images.githubusercontent.com/77539/94067274-035c0100-fdc4-11ea-998a-43702537e9d6.png" />



#### Proposed changelog entry for your changes:

* Add loom.com as a new oembed provider
